compare-oracle-to-fallback/bot-config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I assume that this Bot code probably is just as divergent from our current coding conventions that updating this configuration file and all associated code would not be worth the effort.

Reply:
For this bot and `check-reserve-update-frequency` I don't think it is worth the effort of updating the configuration files/coding convention since they are both very specific to the Aave protocol.